src/Validation.Common.Job/Validation/ValidationRequest.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Services.Validation;
+
+namespace NuGet.Services.Validation
+{
+    public class ValidationRequest : IValidationRequest
```

Comment:
Let's add a comment for when this class will be implemented:

```suggestion
    // TODO: Finalize this class.
    // Tracked by: https://github.com/NuGet/Engineering/issues/3583
    public class ValidationRequest : IValidationRequest
```